tests/test_mle.py

```diff
@@ -317,7 +317,7 @@ def test_infer_node_vals_and_errs_networkx_invariant_wrt_permutation(nx_graph_wi
     g_res = infer_node_vals_and_errs_networkx_partial(g_shuffled, n_bootstrap=2, seed=seed)
 
     for n, (ref_dg, ref_dg_err) in enumerate(zip(ref_dgs, ref_dg_errs)):
-        assert g_res.nodes[n][node_val_prop] == pytest.approx(ref_dg, rel=1e-5)
+        assert g_res.nodes[n][node_val_prop] == pytest.approx(ref_dg, rel=1e-4)
```

Comment:
surprise: for general test flakiness or related to PR?

Reply:
General test flakiness. Never passed locally previously.

Reply:
Interesting that you see different behavior locally (it passes locally for me on master). In any case, I'm in favor of this change as I think the current value is not robust to changing the seed.